src/components/parameters.js

```diff
@@ -635,7 +696,7 @@ const Parameters = ({ showParameters, hideParameters }) => {
                         <MapParameters />
                     </TabPanel>
                     <TabPanel value={tabIndex} index={2}>
-                        {studyUuid && <LoadFlowParameters />}
+                        {studyUuid && <LoadFlow />}
                     </TabPanel>
                     <Grid item xs={12}>
                         <Button
```

Comment:
After a modification in the load flow provider, the loadflow run button does not change (it remains green and we cannot rerun a loadflow, without doing a modification in the study to rerun it again)
See my remark in study-server : https://github.com/gridsuite/study-server/pull/91/files?file-filters%5B%5D=.java&file-filters%5B%5D=.sql&hide-deleted-files=true

Reply:
Fixed in study-server